src/formula_tests.rs

```diff
@@ -71,11 +71,11 @@ fn test_substitute() {
     let a = names.lookup_label(b"cA").unwrap().atom;
     let b = names.lookup_label(b"cB").unwrap().atom;
     assert!(as_str(names.atom_name(subst[a].get_by_path(&[]).unwrap())) == "cadd");
-    assert!(as_str(names.atom_name(subst[a].get_by_path(&[1]).unwrap())) == "c1");
-    assert!(as_str(names.atom_name(subst[a].get_by_path(&[2]).unwrap())) == "c2");
+    assert!(as_str(names.atom_name(subst[a].get_by_path(&[0]).unwrap())) == "c1");
```

Comment:
This code should make use of a simple s-expression printer so that you can do something like `assert_eq!(parsed, "(cadd (c1) (c2))")` here and elsewhere

Reply:
Indeed, and that might have other uses, even if just for debugging.
I'll add that in another PR.

Reply:
See #66